app/views/blog_posts/_case_studies.html.erb

```diff
@@ -0,0 +1,11 @@
+<% case_studies = blog_posts_with_tag('case-study', taggable: taggable) %>
```

Comment:
On having got a feel for this and with us now importing all posts in to the app, I think we should drop the requirement for having a 'case-study' tag for posts to appear in the sidebar. Basically, just have `blog_posts = related_blog_posts(taggable)`, which just gives any blog posts that match the taggable's tags.